src/inmanta/data/__init__.py

```diff
@@ -5208,7 +5217,10 @@ async def create_for_partial_compile(
         the partial compile, i.e. not present in rids_in_partial_compile.
         """
         query = f"""
-            WITH base_version_exists AS (
+            WITH base_version_record AS (
```

Comment:
@arnaudsjs I'm not very confident I did this in a good way, I just jiggled it until it passes the test

Reply:
Looks good to me.